### ISSUE TITLE
Battle for Zendikar

### ISSUE THREAD

**Issue by danieljbrooks** [danieljbrooks/draftsim-launch#9]

**Comment by scottchiefbaker** [comment]
Will you be able to get a full set of card images before launch?

**Comment by danieljbrooks** [comment]
Yes. I've pulled the current cards images from the cockatrice image folder.

We want to update shortly after the full spoiler goes up, so if cockatrice updates slowly, I'll need to download from gatherer or some other online database.

**Comment by scottchiefbaker** [comment]
What URL are you pulling from? I'm not familiar with cockatrice.

**Comment by danieljbrooks** [comment]
Cockatrice downloads card images to a local folder when you mouse over
them. These local folders can then be moved to the website.

Sites like mythicspoiler.com and gatherer host the card images. Downloading
them directly from these sites would be preferred

**Comment by danieljbrooks** [comment]
Hey all, great to have you here. The next week or so is going to hectic with work. I'll be back after that to help improve the site for BFZ.
